### DISCUSSION TITLE
Πρόβλημα με τον grader για το evencyle

### DISCUSSION THREAD

**Discussion by Dim131** [pdp-archive/pdp-archive.github.io#254] [General]
Υπάρχει ένα πρόβλημα με τον grader για το evencyle, καθώς δεν ελέγχει αν είναι οποιοσδήποτε σωστός κύκλος. 

Ελπίζουμε ότι αυτό θα διορθωθεί σύντομα.

**Comment by panospask** [comment]
Νομίζω πως αντίστοιχο πρόβλημα υπάρχει στο 34-marbles.

**Comment by Dim131** [comment]
Για το evencycle, διορθώθηκε τώρα. Για το marbes έφτιαξα το issue #255 να το κοιτάξουμε μέσα στις επόμενες ημέρες
